api/tests/test_token_checker.py

```diff
@@ -169,36 +175,38 @@ def test_is_token_issuer_trusted(self):
         """
         payload_list = []
 
-        # Add a payload without 'iss' field.
-        # to test we reject these as we cannot
-        # tell where it came from (so can't verify it)
+        # Test we reject a payload without 'iss' field
+        # as we cannot tell where it came from (so can't verify it)
         payload_list.append({
             'jti': '098cb343-c45e-490d-8aa0-ce1873cdc5f8',
             'iat': int(time.time()) - 2000000,
             'sub': CLIENT_ID,
-            'exp': int(time.time()) + 200000})
+            'exp': int(time.time()) + 200000}
+        )
```

Comment:
The brace should come down too as the opening one is next to the bracket. See following instance too.

Reply:
ah yes